Automattic/jetpack
# Social: Fix the social icon colours on the admin page

The social icon colours were not being picked up on the admin page, which turned out to be a problem with our Post CSS configuration.

In addition we were using --jp-gray-5 which wasn't defined by the standard theme, but is the same value as --jp-gray. This has been a problem before, so instead of swapping the property name, I've added --jp-gray-5 as an alias.





Fixes Automattic/jetpack-reach#314

### Other information:


## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Add the connection admin feature flag
* Go to the Social Admin page
* Check that any current connections have icons with an appropriate network colour
* Click the 'add connection' button
* Check that the network icons have the correct colours
* Check that the grey borders are still present around the existing connections and the services on the add connection modal

![CleanShot 2024-05-09 at 18 56 20@2x](https://github.com/Automattic/jetpack/assets/96462/5a855448-0aec-4cd1-bafc-2115e8b390d1)

![CleanShot 2024-05-09 at 18 57 24@2x](https://github.com/Automattic/jetpack/assets/96462/33965812-33fb-4be3-8cfd-a8957ff966a5)
